skunkworks_crow/src/main/java/org/odk/share/fragments/FilledFormsFragment.java

```diff
@@ -178,11 +179,16 @@ public void toggle() {
         toggleButtonLabel();
     }
 
+
     private void toggleButtonLabel() {
         if (selectedInstances.size() == instanceAdapter.getItemCount()) {
             toggleButton.setText(getString(R.string.clear_all));
+            instancescount = selectedInstances.size();
+            sendButton.setText("Send(" + (selectedInstances.size() + BlankFormsFragment.filledcount) + ")");
```

Comment:
And always use string resources, don't use hardcoded strings, its easier to change at one place.

Reply:
Yeah, ok but I don't understand why the build is failed. You know the specific reason why?

Reply:
Have you tried running `./gradlew lint checkstyle pmd findbugs` locally.